src/occupational_classification/data_access/soc_data_access.py

```diff
@@ -6,10 +6,33 @@
 import pandas as pd
 
 
+def combine_job_title(row: pd.DataFrame) -> str:
+    """Produces full job title wih IND and ADD qualifiers.
+
+    Args:
+        row (pd.DataFrame): A row containing job title, IND and ADD qualifiers
```

Comment:
The data type should be pd.Series. I've looked it up and it seems like each row in a DataFrame is essentially a Series, where the index corresponds to the column names, and the values are the row's data.